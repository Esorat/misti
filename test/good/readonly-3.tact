fun test() {
    let a: Int = 42; // warning: read-only variable
    if (a == 42) {}
}
