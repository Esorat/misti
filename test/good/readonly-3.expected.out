test/good/readonly-3.tact:2:9:
  1 | fun test() {
> 2 |     let a: Int = 42; // warning: read-only variable
              ^
  3 |     if (a == 42) {}
Read-only variable
Help: Consider creating a constant instead
See: https://nowarp.io/tools/misti/docs/detectors/ReadOnlyVariables